README.rst

```diff
@@ -436,6 +451,28 @@ For manual control over positioning (e.g. for multi-threaded use),
 you may specify `position=n` where `n=0` for the outermost bar,
 `n=1` for the next, and so on.
 
+Writing messages
+~~~~~~~~~~~~~~~~~~~~
+Since ``tqdm`` uses a simple printing mechanism to display progress bars,
+you should not write any message in the terminal using ``print()``.
+
+To write messages in the terminal without any collision with ``tqdm`` bar
+display, a ``.write()`` method is provided:
+
+.. code:: python
+
+    from tqdm import tqdm, trange
+    from time import sleep
+
+    bar = trange(10)
+    for i in bar:
+        # Print using tqdm class method .write()
+        tqdm.write("Done task %i" % i)
+        # Can also use instance method bar.write()
```

Comment:
Checking this evening.

2016-04-22 16:31 GMT+02:00 Casper da Costa-Luis <notifications@github.com>:

> In README.rst <https://github.com/tqdm/tqdm/pull/150#discussion_r60747984>
> :
>
> > +Since ``tqdm`` uses a simple printing mechanism to display progress bars,
> > +you should not write any message in the terminal using ``print()``.
> > +
> > +To write messages in the terminal without any collision with ``tqdm`` bar
> > +display, a ``.write()`` method is provided:
> > +
> > +.. code:: python
> > +
> > +    from tqdm import tqdm, trange
> > +    from time import sleep
> > +
> > +    bar = trange(10)
> > +    for i in bar:
> > +        # Print using tqdm class method .write()
> > +        tqdm.write("Done task %i" % i)
> > +        # Can also use instance method bar.write()
>
> Yes, well, slightly more accurately: a class method called via an
> instance. Will reword that in addition to anything else this weekend.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tqdm/tqdm/pull/150/files/8e1ce8bf117b8b2b7078390aee7488fcd34d053a#r60747984>
>